src/Products/urban/browser/cron/notice.py

```
@@ -148,276 +156,503 @@ def _get_notice_notifications(self):
+    def process(self):
+        if self.licence:
+            self.update_licence()
+        elif self.create_licence_if_missing:
+            self.create_licence()
+        else:
+            raise ValueError(
+                "No licence found with reference number {} / reference FT {} / reference DGATLP".format(
+                    self.notification.reference,
+                    self.notification.referenceFT,
+                    self.notification.referenceDGATLP,
+                )
```

Ruff (0.15.13)

[error] `.format` call has unused arguments at position(s): 2

Remove extra positional arguments at position(s): 2

(F523)